### ISSUE TITLE
arrow-parens with async keyword

### ISSUE THREAD

**Issue by rightaway** [eslint/eslint#4413]
In code like this with eslint 1.9.0 using babel-eslint parser

```js
wrap(async (a, b) => {
  // TODO
});
```

There's a warning on the first line `Expected parentheses around arrow function argument. (arrow-parens)`, even though there are parentheses around the actual arguments `(a, b)`.

This error doesn't appear if I replace `(a, b)` with either `(a)` or `a`.

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by ilyavolodin** [comment]
ESLint doesn't support experimental features directly. As such we can't guarantee that our rules will work with experimental syntax. You might want to file an issue with [eslint-plugin-babel](https://github.com/babel/eslint-plugin-babel) if they do not already support this.
If you can reproduce this case without using async, please reopen this issue.
